src/VisualStudio/Core/Def/RoslynPackage.cs

```diff
@@ -128,6 +132,13 @@ protected override async Task LoadComponentsAsync(CancellationToken cancellation
             LoadComponentsBackgroundAsync(cancellationToken).Forget();
         }
 
+        private void RegisterCodeCleanupProvider()
+        {
+            var regService = ComponentModel.GetService<ICodeCleanUpFixerRegistrationService>();
+            var provider = ComponentModel.GetService<CodeCleanUpFixerProvider>();
+            regService.TryRegisterFixerProvider(provider);
```

Comment:
I feel like this is really odd... Shouldn't 'mef' be used to discover this stuff?

Reply:
This falls under https://github.com/dotnet/roslyn/pull/30039#discussion_r221995049